web/templates/recent_view_filters.hbs

```diff
@@ -1,21 +1,21 @@
 {{> ./dropdown_widget widget_name="recent-view-filter"}}
-<button data-filter="include_private" type="button" class="btn btn-default button-recent-filters {{#if is_spectator}}fake_disabled_button{{/if}}" role="checkbox" aria-checked="true">
+<button data-filter="include_private" type="button" class="btn bootstrap-btn-default button-recent-filters {{#if is_spectator}}fake_disabled_button{{/if}}" role="checkbox" aria-checked="true">
     {{#if filter_pm}}
     <i class="fa fa-check-square-o"></i>
     {{else}}
     <i class="fa fa-square-o"></i>
     {{/if}}
     {{t 'Include DMs' }}
 </button>
-<button data-filter="unread" type="button" class="btn btn-default button-recent-filters {{#if is_spectator}}fake_disabled_button{{/if}}" role="checkbox" aria-checked="false">
+<button data-filter="unread" type="button" class="btn bootstrap-btn-default button-recent-filters {{#if is_spectator}}fake_disabled_button{{/if}}" role="checkbox" aria-checked="false">
     {{#if filter_unread}}
     <i class="fa fa-check-square-o"></i>
     {{else}}
     <i class="fa fa-square-o"></i>
     {{/if}}
     {{t 'Unread' }}
 </button>
-<button data-filter="participated" type="button" class="btn btn-default button-recent-filters {{#if is_spectator}}fake_disabled_button{{/if}}" role="checkbox" aria-checked="false">
+<button data-filter="participated" type="button" class="btn bootstrap-btn-default button-recent-filters {{#if is_spectator}}fake_disabled_button{{/if}}" role="checkbox" aria-checked="false">
```

Comment:
Given these styles are only used for this one component, a good follow-up would be to delete `bootstrap-btn-default` (and remove `bootstrap-btn`) and just copy any useful non-overridden styles onto `button-recent-filters`. Would be a nice cleanup of cruft here.

Reply:
Opened #32406 for the same.